### ISSUE TITLE
[🐛] Impossible to install npm snyk 1.441.0 globally

### ISSUE THREAD

**Issue by vulcaryn** [snyk/snyk#1613]
- `node -v`: 10.13.0
- `npm -v`: 6.4.1
- `snyk -v`: 1.441.0
- OS: Linux (Circle CI pipeline)
- Command run: sudo npm i -g snyk

### Expected behaviour
Snyk installed successfully

### Actual behaviour
Try with latest version (not specified)
```
66dde:~/test$ sudo npm i -g snyk
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git clone --depth=1 -q -b master https://github.com/open-policy-agent/npm-opa-wasm.git /root/.npm/_cacache/tmp/git-clone-fdfd5ad4
npm ERR! fatal: could not create leading directories of '/root/.npm/_cacache/tmp/git-clone-fdfd5ad4': Permission denied
npm ERR!

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-04T15_00_32_770Z-debug.log
```

Try with latest version (specified)
```
66dde:~/test$ sudo npm i -g snyk@1.441.0
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git clone --depth=1 -q -b master https://github.com/open-policy-agent/npm-opa-wasm.git /root/.npm/_cacache/tmp/git-clone-c4e375bf
npm ERR! fatal: could not create leading directories of '/root/.npm/_cacache/tmp/git-clone-c4e375bf': Permission denied
npm ERR!

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-04T15_00_45_794Z-debug.log
```

Try with the previous version (specified)
```
66dde:~/test$ sudo npm i -g snyk@1.440.5
/usr/local/bin/snyk -> /usr/local/lib/node_modules/snyk/dist/cli/index.js
+ snyk@1.440.5
added 453 packages from 333 contributors in 15.467s
```
### Steps to reproduce
Just try a `npm i -g snyk`

**Comment by JackuB** [comment]
Yes, you are right, we are reverting that change

**Comment by JackuB** [comment]
Hi, sorry for this! Is the issue solved for you in the latest `snyk@1.443.0`?

We'll be adding more checks to prevent this in the future

**Comment by vulcaryn** [comment]
Thanks for your quick answer & fix 👌
